.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -175,6 +175,9 @@ jobs:
         run: |-
           poetry run rasa --version
 
+          # Run GPUs in a deterministic mode
+          export TF_DETERMINISTIC_OPS=1
```

Comment:
Can we move to the `env` a few lines above?

Reply:
Moved!